src/Translate/languages/fr.json

```diff
@@ -1,26 +1,26 @@
 	"Successfully started Tor onion hidden services.": "Tor activé avec succès.",
-	"Unable to start hidden services, please check your config.": "Incapable d'activer le service Tor, veuillez vérifier votre configuration.",
-	"For faster connections open <b>{0}</b> port on your router.": "Pour une meilleure connection, ouvrez le port <b>{0}</b> sur votre routeur.",
-	"Your connection is restricted. Please, open <b>{0}</b> port on your router": "Votre connection est limitée. Veuillez ouvrir le port <b>{0}</b> sur votre routeur",
-	"or configure Tor to become full member of ZeroNet network.": "ou configurez Tor afin qu'il fasse partie intégrante de ZeroNet.",
+	"Unable to start hidden services, please check your config.": "Incapable d'activer Tor, veuillez vérifier votre configuration.",
+	"For faster connections open <b>{0}</b> port on your router.": "Pour une meilleure connectivité, ouvrez le port <b>{0}</b> sur votre routeur.",
+	"Your connection is restricted. Please, open <b>{0}</b> port on your router": "Connectivité limitée. Veuillez ouvrir le port <b>{0}</b> sur votre routeur",
+	"or configure Tor to become full member of ZeroNet network.": "ou configurez Tor afin d'avoir accès aux pairs ZeroNet Onion.",
 
 	"Select account you want to use in this site:": "Sélectionnez le compte que vous voulez utiliser pour ce site:",
 	"currently selected": "présentement sélectionné",
 	"Unique to site": "Unique au site",
 
 	"Content signing failed": "Échec à la signature du contenu",
-	"Content publish queued for {0:.0f} seconds.": "Mise en attente de la publication du contenu pour {0:.0f} secondes.",
-	"Content published to {0} peers.": "Contenu publié pour {0} pairs.",
+	"Content publish queued for {0:.0f} seconds.": "Publication du contenu différée {0:.0f} secondes.",
+	"Content published to {0} peers.": "Contenu publié à {0} pairs.",
 	"No peers found, but your content is ready to access.": "Aucun pair trouvé, mais votre contenu est accessible.",
-	"Your network connection is restricted. Please, open <b>{0}</b> port": "Votre connection est limitée. Veuillez ouvrir le port <b>{0}</b>",
+	"Your network connection is restricted. Please, open <b>{0}</b> port": "Connectivité limitée. Veuillez ouvrir le port <b>{0}</b>",
 	"on your router to make your site accessible for everyone.": "sur votre routeur pour que votre site soit accessible à tous.",
 	"Content publish failed.": "Échec de la publication du contenu.",
-	"This file still in sync, if you write it now, then the previous content may be lost.": "Ce fichier est encore en synchronisation, si vous l'enregistrez maintenant le contenu antérieur risque d'être effacé.",
-	"Write content anyway": "Enregistrer le contenu",
-	"New certificate added:": "Ajout du nouveau certificat:",
+	"This file still in sync, if you write it now, then the previous content may be lost.": "Ce fichier n'est pas à jour, si vous le modifiez maintenant une version antérieure pourrait être perdue.",
```

Comment:
Does it mean that the changes would be lost because there is an earlier version shared on the network that could overwrite this one?